.github/workflows/php.yml

```
@@ -28,12 +29,17 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install libmemcached
+      run: sudo apt-get install -y libmemcached-dev
+
     - name: Install PHP with extensions
-      uses: shivammathur/setup-php@v2
+      uses: shivammathur/setup-php@verbose
```

Looks like you changed this for debugging, do we want to switch this back to `v2`?
```suggestion
      uses: shivammathur/setup-php@v2
```

---

Already on it. Rebase was just pushed